content/blog/2019/08/2019-08-17-remoting-kafka-kubernetes-release-2.0.adoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Remoting over Apache Kafka plugin release 2.0"
+tags:
+- remoting
+- kafka
+- kubernetes
+- helm
+- developer
+- gsoc
+- gsoc2019
+author: longnguyen
+---
+
+= Remoting over Apache Kafka plugin release 2.0
+
+I am Long Nguyen from link:https://international.fpt.edu.vn/[FPT University], Vietnam. My project for Google Summer of Code 2019 is link:https://jenkins.io/projects/gsoc/2019/remoting-over-apache-kafka-docker-k8s-features/[Remoting over Apache Kafka with Kubernetes features]. After a successful link:https://jenkins.io/blog/2019/07/11/remoting-kafka-kubernetes-phase-1/[Phase 1], finally the 2.0 version of the plugin has been released. The 2.0 version provides seamless integration with Kubernetes environment.
+
```

Comment:
Would be nice to have few sentences describing the plugin and the problem you were trying to solve. Benefits to the community and so on